lib/install/validate/storage.go

```diff
@@ -32,14 +33,26 @@ func (v *Validator) storage(ctx context.Context, input *data.Data, conf *config.
 	defer trace.End(trace.Begin(""))
 
 	// Image Store
-	imageDSpath, ds, err := v.DatastoreHelper(ctx, input.ImageDatastoreName, "", "--image-datastore")
+	imageDSpath, ds, err := v.DatastoreHelper(ctx, input.ImageDatastorePath, "", "--image-datastore")
+
+	// provide a default path if only a DS name is provided
+	if imageDSpath.Path == "" {
+		imageDSpath.Path = path.Join(input.DisplayName, "images")
```

Comment:
This is the same behavior as before, I believe. I did not intentionally change the behavior. There should be logic to make sure the directory doesn't exist in the routine that creates the directory (regardless of its location) but I will double-check